package.json

```diff
@@ -117,7 +118,7 @@
     "fixturez": "^1.1.0",
     "flushable": "^1.0.0",
     "fs-extra": "^7.0.0",
-    "gatsby-mdx": "^0.4.4",
+    "gatsby-mdx": "^0.6.3",
```

Comment:
This updates `mdx` to v1. This PR takes care of the breaking changes mentioned in [their migration guide](https://mdxjs.com/migrating/v1)